AlpsSDK/ProximityHandler.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  ProximityHandler.swift
+//  AlpsSDK
+//
+//  Created by Wen on 26.10.17.
+//  Copyright © 2017 Alps. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+import Alps
+
+protocol ProximityHandlerDelegate : class {
+    func didRangeBeacons(manager: ContextManager, beacons: [CLBeacon], knownBeacons: [IBeaconTriple])
+}
+
+class ProximityHandler: NSObject, ProximityHandlerDelegate {
+    var refreshTimer: Int = 60 * 1000 // timer is in milliseconds
+    var beaconsDetected: [String: [IBeaconTriple]] = [:]
+    var beaconsTriggered: [String: [String: ProximityEvent]] = [:]
+    
+    override init() {
+        super.init()
+        beaconsTriggered["unknown"] = [String: ProximityEvent]()
```

Comment:
it's better to use enum's rather than hardcoded strings as keys.

Reply:
ok on it !